boto/iam/connection.py

```diff
@@ -1102,7 +1102,7 @@ def create_instance_profile(self, instance_profile_name, path=None):
             params['Path'] = path
         return self.get_response('CreateInstanceProfile', params)
 
-    def _build_policy(self, assume_role_policy_document=None):
+    def build_policy(self, assume_role_policy_document=None):
```

Comment:
If we publish this method lets update the docstring for it.

```
Build default policy document when `assume_role_policy_document` is not specified.

:type assume_role_policy_document: str
:return: Serialized to string JSON document of the policy
:rtype: str
```

**Not sure the docstring is complete.